src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Refactoring/RenameEndpoint.cs

```diff
@@ -142,10 +137,11 @@ protected override Task<WorkspaceEdit> HandleDelegatedResponseAsync(WorkspaceEdi
             };
         }
 
-        private async Task<List<DocumentSnapshot?>> GetAllDocumentSnapshotsAsync(DocumentSnapshot skipDocumentSnapshot, CancellationToken cancellationToken)
+        private async Task<List<DocumentSnapshot?>> GetAllDocumentSnapshotsAsync(DocumentContext skipDocumentContext, CancellationToken cancellationToken)
```

Comment:
What does "skip" mean in this context? I know you're following the previous naming scheme but it seems weird to me as someone without domain knowledge here.

Reply:
So firstly, I have just as much "knowledge" as you, and I'm totally just maintaining the old naming, but this is going through and finding all of the places where the rename needs to be handled, but since we've already processed the file that the user is executing the rename from, it skips that one, and just returns the rest. The comment on line 155 explains it I guess.